src/test/txindex_tests.cpp

```diff
@@ -53,7 +53,7 @@ BOOST_FIXTURE_TEST_CASE(txindex_initial_sync, TestChain100Setup)
         CScript coinbase_script_pub_key = GetScriptForDestination(coinbaseKey.GetPubKey().GetID());
         std::vector<CMutableTransaction> no_txns;
         const CBlock& block = CreateAndProcessBlock(no_txns, coinbase_script_pub_key);
-        BOOST_CHECK_EQUAL(block.proof.size(), Params().GetSignedBlocksCondition().threshold);
+        BOOST_CHECK_EQUAL(block.proof.size(), 64);
```

Comment:
Not use SCHNORR_SIGNATURE_SIZE constant value?

Reply:
done.